instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/config/ConfigParsingException.java

```diff
@@ -6,6 +6,8 @@
 package io.opentelemetry.instrumentation.api.config;
 
 public class ConfigParsingException extends RuntimeException {
+  private static final long serialVersionUID = 384182080843382041L;
```

Comment:
I would use `serialVersionUID = 1L;` otherwise it is hard to tell whether this is just a random number or deliberately computed with serialver